example.properties

```diff
@@ -0,0 +1,64 @@
+# Common options
+error.maxRedeliveries=5
+jms.brokerUrl=tcp://localhost:61616
+jms.username=
+jms.password=
+jms.connections=10
+
+# Custom Http client options
+# All timeouts in milliseconds
+request.configurer.enabled=false
+request.timeout=-1
+connection.timeout=-1
+socket.timeout=-1
+
+# Solr indexer options
```

Comment:
Should this be "Fedora indexer options"?

Reply:
DOH